.github/workflows/pre-release.yml

```diff
@@ -120,9 +120,9 @@ jobs:
 
             > :warning: **WARNING:** 
             >
-            > This pull request has been labeled as `automerge` and will be merged automatically after the checks are completed. **Do not merge manually if any check fails**.
+            > This pull request should be auto-merged. **Do not merge manually if any check fails**.
             >
-            > Instead, remove the `automerge` label and push your fixes to the [`pre-release-${{ env.DATE }}`](https://github.com/tardis-bot/tardis/tree/pre-release-${{ env.DATE }}) branch on [**tardis-bot/tardis**](https://github.com/tardis-bot/tardis).
+            > Instead, disable auto-merge and push your fixes to the [`pre-release-${{ env.DATE }}`](https://github.com/tardis-bot/tardis/tree/pre-release-${{ env.DATE }}) branch on [**tardis-bot/tardis**](https://github.com/tardis-bot/tardis).
```

Comment:
How do we disable auto-merge?

Reply:
At the bottom of the PR, It's quite visible
![image](https://user-images.githubusercontent.com/22769314/173593307-2c89b0cb-f556-4aa9-91d3-eab4d90bb08a.png)